app/utils/cyaHelpers/SummaryListRowHelper.scala

```diff
@@ -55,6 +55,8 @@ private[utils] class SummaryListRowHelper(implicit messages: Messages) {
 
   protected def formatAsCountry(country: Country): Content = country.description.toText
 
+  protected def formatAsPackage(packageType: PackageType): Content = packageType.toString.toText
```

Comment:
Does this not just do the same thing as `formatAsText`?

Reply:
Yea they passed, I'll switch and push up